src/components/common/request-details.vue

```diff
@@ -67,28 +67,20 @@
 </template>
 
 <script lang="ts">
-import { ApplicantI } from '@/interfaces'
+import { ApplicantI, NameChoicesIF } from '@/interfaces'
 import { Component, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
 
 @Component({})
 export default class RequestDetails extends Vue {
-  @Prop(Object)
-  readonly applicant: ApplicantI
-
-  @Prop(Array)
-  readonly nameChoices: {
+  @Prop() applicant: ApplicantI
```

Comment:
Please restore the "readonly" prefix on the props.

Reply:
Oops, will do!